tuxemon/menu/menu.py

```diff
@@ -17,6 +17,7 @@
 from tuxemon.graphics import ColorLike
 from tuxemon.platform.events import PlayerInput
 from tuxemon.animation import Animation
+from tuxemon.states.transition.fade import FadeInTransition
```

Comment:
what is `FadeInTransition` used?

Reply:
Sorry, I tried doing a fade transition when the player goes back to the start menu. It didn't work out.

Reply:
For fade states, push the next state, then push the fade state.